static/src/javascripts/projects/common/views/international-message.html

```diff
@@ -3,6 +3,10 @@
 </p>
 <ul class="site-message__actions u-unstyled">
     <li class="site-message__actions__item">
+        <i class="i i-arrow-white-right"></i>
+        <a href="http://www.theguardian.com/help/insideguardian/2015/apr/20/giving-our-international-readers-a-fresh-take-on-the-news" data-link-name="blog">Find out more</a>
```

Comment:
Hey, what about our old friend `LinkTo(/help/insideguardian/2015/apr/20/giving-our-international-readers-a-fresh-take-on-the-news)`